<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="gameGraphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>660</y>
      <width>108</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="turnLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TURN :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="surrenderButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Surrender</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>660</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="redPiecesLabel">
       <property name="text">
        <string>RED: 15 Pieces Remaining</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blackPiecesLabel">
       <property name="text">
        <string>BLACK: 15 Pieces Remaining</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="redWinsLabel">
       <property name="text">
        <string>RED: 0 Wins</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="blackWinsLabel">
       <property name="text">
        <string>BLACK: 0 Wins</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
